packages/neptune-extension/src/common/components/notifications/NotificationsContainer.tsx

```diff
@@ -17,15 +18,21 @@ const NotificationsContainer: React.FC = () => {
   return (
     <NotificationsPortal>
       {
-        notifications.map((notification, key) => (
-          <Toast
-            key={key}
-            onClose={() => (deleteNotification(notification.id))}
-            type="success"
-          >
-            {JSON.stringify(notification, null, 2)}
-          </Toast>
-        ))
+        notifications.map((notification, key) => {
```

Comment:
All these changes will be removed before merging.
It was needed to test if notification is shown and works properly (ie. use can click on a link to Neptune App)